plugins/entity-validation/src/components/EntityValidationPage/EntityValidationPage.tsx

```diff
@@ -83,12 +83,7 @@ export const EntityValidationPage = (props: {
           <Grid item md={3} xs={12}>
             <Grid container alignItems="center" style={{ height: '100%' }}>
               <Grid item>
-                <Button
-                  style={{ textDecoration: 'none' }}
-                  variant="outlined"
-                  to="#"
-                  onClick={parseYaml}
-                >
+                <Button size="large" onClick={parseYaml}>
```

Comment:
I wouldn't switch to the "material-ui/core" `Button`. The desired is using the internal backstage Buttons and Links, as they can be used for metrics and other stuff. Also, note that now `Button` is deprecated, `LinkButton` should be used instead.

Reply:
I was going to switch it to a LinkButton, but since it didn't have a "to" and wasn't really a link, thought about going this direction.. but stuff about the metrics and things is a fair statement. I can make that change back.

Reply:
updated to use LinkButton